subsys/compress/include/nrf_compress/implementation.h

```diff
@@ -0,0 +1,74 @@
+/*
```

Comment:
The include file is placed incorrectly. It should be located in `ncs/nrf/include`, to follow the pattern of other subsys libraries.